tests/Hyperwallet/Tests/HyperwalletTest.php

```diff
@@ -3549,4 +3570,122 @@ public function prepaidCardStatusTransitionProvider() {
             'unlockPrepaidCard' => array('unlockPrepaidCard', PrepaidCardStatusTransition::TRANSITION_UNLOCKED)
         );
     }
+
+
+    public function testUpdateVerificationStatus_allParameters() {
+
+        // Setup
+        $client = new Hyperwallet('test-username', 'test-password');
+        $apiClientMock = $this->createAndInjectApiClientMock($client);
+        $user = new user(array('verificationStatus'=> User::VERIFICATION_STATUS_REQUESTED));
+        \Phake::when($apiClientMock)->doPut('/rest/v3/users/{user-token}', array('user-token' => 'test-user-token'), $user, array())->thenReturn(array("status"=> User::STATUS_PRE_ACTIVATED, 'verificationStatus'=> User::VERIFICATION_STATUS_REQUIRED));
+        \Phake::when($apiClientMock)->doGet('/rest/v3/users/{user-token}', array('user-token' => 'test-user-token'), array())->thenReturn(array("status"=> User::STATUS_PRE_ACTIVATED, 'verificationStatus'=> User::VERIFICATION_STATUS_REQUIRED));
+
+        // Run test
+        try {
+            $responseUser = $client->updateVerificationStatus('test-user-token', User::VERIFICATION_STATUS_REQUESTED);
+        } catch (HyperwalletArgumentException $e) {
+            $this->assertEquals('userToken is required!', $e->getMessage());
+        }
+        $this->assertNotNull($responseUser);
+        $this->assertEquals('REQUIRED', $responseUser->getVerificationStatus());
+        // Validate mock
+        \Phake::verify($apiClientMock)->doPut('/rest/v3/users/{user-token}', array('user-token' => 'test-user-token'), $user, array());
+    }
+
+
+    //IT test cases--Please uncomment the following lines
+
+    public function testRequestUserVerificationWithAndWithoutRoleIT() {
+        $username = "selrestuser@1861681";
+        $password = "Password1!";
+        $programToken = "prg-eedaf875-01f1-4524-8b94-d4936255af78";
+        $userToken = "usr-a1a77c00-e2af-41c4-b0c8-f85097964ae1";
+        $server = "https://localhost-hyperwallet.aws.paylution.net:8181";
```

Comment:
change the server url since this is for v3. Change it for other cases too.

Reply:
Done